__mocks__/react-i18next.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { translate, translateWithFormatting } from '@/utils/test/i18n';
+import React, { FunctionComponent } from 'react';
+
+const Trans: FunctionComponent<{ i18nKey: string }> = ({
+  i18nKey,
+  children,
+}) => <div>{translateWithFormatting(i18nKey, children)}</div>;
+
+const useTranslation = () => {
+  return { t: translate };
```

Comment:
any reason why we could not export `translate` as translate? I think the name should at least reflect what `t` is/is for